encryption/encryption.go

```diff
@@ -57,9 +86,17 @@ func decryptPayloadPath(jsonPayload *gabs.Container, jsonPathIn string, jsonPath
 	if jsonPathOut == "$" {
 		jsonPayload = jsonDecryptedPayload
 	} else {
-		jsonPayload.DeleteP(jsonPathIn)
-		jsonPayload.Set(jsonDecryptedPayload, jsonPathOut)
+		if jsonPathOut[0] == '$' {
```

Comment:
this If statement is duplicate, please consider to implement a helper function for it